### PR TITLE
TMS-666: Single blog

### PR DESCRIPTION
**Pull request by vhirvonen**
CR: https://geniemdev.atlassian.net/browse/TMS-627

## Description
Single blog & comments

## Motivation and Context
TMS-666

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+- TMS-666: Single blog #60
```
Comment (ville6000) [inline CHANGELOG.MD:12]: Nice

File: assets/styles/settings/brand/_typography.scss [hunk 2]
```diff
@@ -138,6 +138,36 @@ h6,
     }
 }
 
+.has-text-align-left {
```
Comment (ville6000) [inline assets/styles/settings/brand/_typography.scss:141]: https://bulma.io/documentation/helpers/typography-helpers/#responsive-alignment nämä tulee Bulmasta jo mukana

**Review by ville6000 — APPROVED** [review]

File: CHANGELOG.MD (hunk 1 above)
Comment (vhirvonen) [reply CHANGELOG.MD:12]: 😈

File: assets/styles/settings/brand/_typography.scss (hunk 2 above)
Comment (vhirvonen) [reply assets/styles/settings/brand/_typography.scss:141]: ✅

**Pull request merged**